src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -568,7 +569,7 @@ case class JesBackend(actorSystem: ActorSystem)
       status match {
         case Run.Success(events) if backendCall.runtimeAttributes.failOnStderr && stderrLength.intValue > 0 =>
           // returnCode will be None if it couldn't be downloaded/parsed, which will yield a null in the DB
-          FailedExecutionHandle(new Throwable(s"${log.tag} execution failed: stderr has length $stderrLength"), returnCode.toOption).future
+          FailedExecutionHandle(new Throwable(s"${log.tag} execution failed: stderr has length $stderrLength"), returnCode.toOption map ScriptReturnCode).future
```

Comment:
Could you somehow functionify `returnCode.toOption map ScriptReturnCode` and use it here and below. Personally I'd love to see it writable as `returnCode.toScriptReturnCode` or something like that but YMMV